tests/test_invenio_records_files.py

```diff
@@ -27,14 +27,19 @@
 
 from __future__ import absolute_import, print_function
 
+import mock
 import pytest
+from flask import url_for
 from invenio_files_rest.errors import InvalidOperationError
 from invenio_files_rest.models import Bucket, ObjectVersion
+from invenio_files_rest.views import blueprint as files_rest_blueprint
```

Comment:
can we move it to `tests/conftest.py` together with `app.register_blueprint(files_rest_blueprint)`?